tests/integration/conftest.py

```diff
@@ -16,17 +16,24 @@
 import pexpect
 import pytest
 
-from envparse import env
+from dotenv import dotenv_values
 
 
 try:
     from pathlib import Path
 except ImportError:
     from pathlib2 import Path
 
-logging.basicConfig(level="DEBUG" if env.str("DEBUG") else "INFO", stream=sys.stderr)
+# Commenting out this line as we do not have any .env file in the repo
+# also envparse (lib with env.read_envfile) was last updated 2015 so this is obsolete
+# if we need load envvars from a .env file in the future there is maintained python-dotenv
+# env.read_envfile(str(Path(__file__).parents[2] / ".env"))
+
+logging.basicConfig(level="DEBUG" if os.environ.get("DEBUG") else "INFO", stream=sys.stderr)
```

Comment:
Would that work instead? 
```suggestion
logging.basicConfig(level=os.environ.get("DEBUG", "INFO"), stream=sys.stderr)
```

Reply:
+1

Reply:
Good point, thanks @r0x0d !